server/webhook_worker.go

```diff
@@ -45,10 +45,6 @@ func (ww webhookWorker) process(msg *webhookMessage) (err error) {
 	}
 
 	v := wh.(*webhook)
-	if err = v.JiraWebhook.expandIssue(ww.p, msg.InstanceID); err != nil {
```

Comment:
are we 100 percent sure this function is not needed for all 3 cases of authentication?